configs/domain/index.ts

```diff
@@ -0,0 +1,21 @@
+import {getDenominationSymbols} from "./get-denomination-symbols";
+import {getLendingProtocols} from "./get-lending-protocols";
+import {getNetworks} from "./get-networks";
+import {getTokensByNetwork} from "./get-tokens-by-network";
+import {getTokensByTokenSymbol} from "./get-tokens-by-token-symbol";
+
+export default function () {
+    /**
+     * Goal is to capture domain specific configurations that is common
+     * across all Summer.fi systems and apps.
+     *
+     * Possible extensions could include Products / Product categories / Strategies etc
+     */
+    return {
+        tokens: getTokensByTokenSymbol(),
+        tokensByNetwork: getTokensByNetwork(),
+        lendingProtocols: getLendingProtocols(),
+        networks: getNetworks(),
```

Comment:
and this is networkNameByNetwork
![image](https://github.com/OasisDEX/oazo-configuration/assets/739075/d1cedf38-6337-4f31-94c7-4fc18412ac3a)